README.md

```diff
@@ -24,9 +24,9 @@ Contains the user manual for the given repo, so how the user can start the given
 
 Contains protocols of meetings regarding the given repo/ concept for the given day. Filenames are in the format `YYYY-mm-dd-protocol.md`.
 
-### [`/internal`](internal/README.md)
+### [`/test-plans`](test-plans)
```

Comment:
See my comment on the issue.
I think it makes the most sense to see them as part of the dev-manuals, that's where they are most logical.